perma_web/perma/celery_tasks.py

```diff
@@ -1224,3 +1229,73 @@ def conditionally_queue_internet_archive_uploads_for_date_range(start_date_strin
 
     else:
         logger.info("Skipped the queuing of file upload tasks: max tasks already in progress.")
+
+
+# WACZ CONVERSION
+
+def format_time(seconds_val):
+    """
+    Converts seconds to minutes
+    """
+    if seconds_val >= 60:
+        return f"{round(seconds_val / 60, 1)} minutes"
+    else:
+        return f"{math.ceil(seconds_val)} seconds"
+
+
+@shared_task
+def convert_warc_to_wacz(input_path, output_folder, benchmark_log):
+    """
+    Converts WARC file to WACZ
+    Saves file in a temp folder
+    If successful, saves file in storage
+    """
+    start_time = time.time()
+    input_file_name = input_path.split('/')[-1]
+    output_path = f"{output_folder}/{input_file_name.split('.')[0]}.wacz"
+    exception_occurred = False
+    error_output = ''
+
+    try:
+        subprocess.run(["npx", "js-wacz", "create", "-f", input_path, "-o", output_path],
+                                      capture_output=True, check=True, text=True)
+    except subprocess.CalledProcessError as e:
+        exception_occurred = False
```

Comment:
```suggestion
        exception_occurred = True
```
I think this still needs a change.

Reply:
Yes, thank you @bensteinberg, missed this!